look/example/lib/ui/common/look/look_data/specific_look_data/color_look_data.dart

```diff
@@ -0,0 +1,91 @@
+import 'package:flutter/material.dart';
+
+class ColorLookData {
+  const ColorLookData({
+    required this.brightness,
+    required this.primary,
+    required this.primaryContainer,
+    required this.primaryPressed,
+    required this.primaryDisabled,
+    required this.secondary,
+    required this.secondaryContainer,
+    required this.secondaryPressed,
+    required this.secondaryDisabled,
+    required this.tertiary,
+    required this.tertiaryDisabled,
+    required this.onPrimary,
+    required this.onSecondary,
+    required this.error,
+    required this.neutral,
+    required this.onError,
+    required this.background,
+    required this.onBackground,
+    required this.surface,
+    required this.onSurface,
+    required this.white10p,
+    required this.black10p,
+    required this.overlay,
+    required this.green,
+    required this.lime,
```

Comment:
Let's keep here only 12 of the material color scheme colors that are required:

primary,
primaryVariant,
secondary,
secondaryVariant,
onPrimary,
onSecondary,
background,
surface,
onBackground,
onSurface,
error,
onError

All others are optional or dependent on the project